src/Ninja/G1CP/Content/Fixes/Session/fix042_GuardExitDialog.d

```diff
@@ -35,55 +11,26 @@ func int G1CP_042_ConfirmByteCode(var int funcId) {
  * present) is replaced to check if there is another exit dialog.
  */
 func int G1CP_042_GuardExitDialog() {
-    var int applied1; applied1 = FALSE;
-    var int applied2; applied2 = FALSE;
+    var int applied; applied = FALSE;
```

Comment:
The variable is no longer necessary. Since it is only one check, it can be handled by returning true from inside the if-condition and false otherwise.